src/Sulu/Bundle/ContactBundle/Controller/ContactController.php

```diff
@@ -34,6 +36,8 @@
  */
 class ContactController extends RestController implements ClassResourceInterface, SecuredControllerInterface
 {
+    use IdsHandlingTrait;
```

Comment:
Why do you need this in the `ContactController`? It should only return a single type of entities anyway, shouldn't it?